src/central-bank-digital-currency-dodao-academy.yaml

```diff
@@ -316,6 +315,39 @@ steps:
     name: Challenges with CBDCS
     stepItems: []
     uuid: a6f16568-6ce0-4189-8135-b99e69782b03
+  - content: ''
+    name: Evaluation
+    stepItems:
+      - answerKeys:
+          - C
+          - D
+        choices:
+          - content: Not backed by any valuable asset
+            key: A
+          - content: Increase banking frauds
+            key: B
+          - content: Technologically demanding
+            key: C
+          - content: Government Controlled (Centralised)
+            key: D
+        content: What are the most common issues with CBDC?
+        type: SingleChoice
+        uuid: 98dd0750-9388-44c6-86f2-64837758777e
+      - answerKeys:
+          - B          
+        choices:
+          - content: Hacking 
+            key: A
+          - content: Interconversion into CBDC of respective nationalities
```

Comment:
`Interconversion into CBDC of respective nationalities`  an issue?

Reply:
I have re phrased it